Sources/MapboxDirectionsCLI/CodingOperation.swift

```diff
@@ -89,21 +131,33 @@ class CodingOperation<ResponceType : Codable, OptionsType : DirectionsOptions >
     
     func execute() throws {
         
-        let input = FileManager.default.contents(atPath: NSString(string: options.inputPath).expandingTildeInPath)!
+//        let input = FileManager.default.contents(atPath: NSString(string: options.inputPath).expandingTildeInPath)!
         let config = FileManager.default.contents(atPath: NSString(string: options.configPath).expandingTildeInPath)!
+        let input: Data!
```

Comment:
It looks like this variable is set in both branches below, and it’s only ever read by an `if let` statement, so it might not need to be implicitly unwrapped.